packages/cli/src/config.ts

```diff
@@ -7,7 +7,7 @@ export const dependencies: string[] = [
 
 export const isLocal = process.env.NODE_ENV === 'local';
 export const daemonPackage = isLocal ? path.join(__dirname, '../../daemon') : '@pipcook/daemon';
-export const boardPackage = isLocal ? path.join(__dirname, '../../pipboard') : '@pipcook/pipboard';
+export const boardPackage = '@pipcook/pipboard';
```

Comment:
Should we also delete isLocal mode for daemon above?

Reply:
Yea, we are able to remove local mode absolutely. /cc @FeelyChau

Reply:
It's done by #472